### ISSUE TITLE
python crashes when switching characters in character editor (running from source only, executable not affected)

### ISSUE THREAD

**Issue by blitzmann** [pyfa-org/Pyfa#1421]
Reported by @Neugeniko. I can reproduce this on one of my laptops, but not on my home machine. Open Character editor, and switch a character. There's a hard crash, no tracback. :/ could be due to a package mismatch on the two installations of python, would have to compare side by side

**Comment by Neugeniko** [comment]
Could be related to the skills tab as with another tab selected the crash doesn't happen when switching a character.

**Comment by Neugeniko** [comment]
Line 533 in characterEditor.py where we populate the tree.

childId = tree.AppendItem(root, name, imageId, data=('group', id))

But for some reason it doesnt fail on initialization just on subsequent calls.

**Comment by Neugeniko** [comment]
Not having much progress on this. Would like to get it out of the way so I can do another pass for bugs since the bugs from my first pass have been mostly fixed.

**Comment by Neugeniko** [comment]
This is my output from pip freeze.

certifi==2018.1.18
chardet==3.0.4
cycler==0.10.0
idna==2.6
Logbook==1.2.1
markdown2==2.3.5
matplotlib==2.1.2
numpy==1.14.1
packaging==16.8
pyparsing==2.2.0
python-dateutil==2.6.1
pytz==2018.3
requests==2.18.4
roman==2.0.0
six==1.11.0
SQLAlchemy==1.2.3
urllib3==1.22
wxPython==4.0.1

**Comment by blitzmann** [comment]
Yep, going to look into this one more closely today. Thansk for the pip freeze, will check what differences I have

**Comment by blitzmann** [comment]
Seems to be an issue going from wxPython 4.0.0b2 to 4.0.0 (4.0.1 still has it as well). I'm going to try to create a small test case for this and submit it as a wxpython bug (if other investigation doesn't yield anything)

**Comment by blitzmann** [comment]
Tracking on https://github.com/wxWidgets/Phoenix/issues/767

For now, the only workaround is to drop down to 4.0.0b2, which should be fine since b2 was pretty stable (executable is using b2, hence why this isn't a problem in the build)

**Comment by blitzmann** [comment]
As a temp measure for now, for now, I will set the requirement to b2. If this is an issue on the wxWidgets level, it might take longer to fix, and then we'd have to wait for a new wxPython release as well.

**Comment by Neugeniko** [comment]
Thanks for the workaround. I'll be doing a another pass for bugs and a little more detailed look at preference bugs the next couple days.

**Comment by blitzmann** [comment]
> preference bugs

I didn't know there were bugs with preferences >.<

**Comment by blitzmann** [comment]
wxPython 4.0.2 has been released, which makes this issue resolved! However, will keep it open for now until I can do a bit more testing on this version (there's a few other fixes, specifically for events I believe).
